goerli/nodekey

```diff
@@ -0,0 +1 @@
+5622b144abcb8a29097a95e1c53b4eb880f2afd5e127c2921eafb13a56c30139
```

Comment:
Don’t commit database to git please

Reply:
https://git-scm.com/book/en/v2

Reply:
I am sorry. I have removed the commits with the db.